src/App.js

```diff
@@ -1,21 +1,27 @@
 import React from 'react';
 import './App.css';
 import { BrowserRouter as Router, Routes, Route, BrowserRouter } from "react-router-dom";
-
+import { categoryData } from '../src/DummyData/sampleCategoryData.js'
 
 import LandingPage from './Components/LandingPage';
 import CameraCapture from './Components/CameraCapture';
 import Leaderboard from './Components/Leaderboard';
+import CategoryDescription from './Components/CategoryDescription';
+import LearnMore from './Components/LearnMore';
 
 const App = () => {
   return (
```

Comment:
this component is very high up in the root and difficult to unti test out at times so no test needed for this component.